api/src/wfl/source.clj

```diff
@@ -147,16 +157,37 @@
                                   column dataset)
       (assoc source :dataset dataset))))
 
+(defn combine-tdr-source-details
+  "Reduce to `result` by combining the Terra Data Repo source `_detail`."
```

Comment:
This docstring doesn't mean anything to me  - I'm none-the-wiser about what this function does

Reply:
Howzabout?

"Reduce to `result` by combining the Terra Data Repo source `_detail`.
  Collect `datarepo_row_ids` already seen while preserving the latest
 `end_time` and the earliest `start_time`."